demos/fleet_provisioning/fleet_provisioning_with_csr/claim_policy.json

```diff
@@ -0,0 +1,31 @@
+ {
```

Comment:
The name of the file can contain `example_` prefix to clearly suggest that this is an example template